client/site-profiler/hooks/use-delayed-query-time.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react';
+
+// Used to generate a different text for the "Check site" button
+export function useDelayedQueryTime( domain: string, isFetching: boolean, limit = 3000 ) {
```

Comment:
I suggest a different naming here.

It is not clear enough, what the hook is supposed to do. Looking into logic, there is no "delaying" logic.
It's only supposed to count a time after the limit we provide.

Maybe `useLongFetchingDetection`, or something similar.

Do you agree?

Reply:
Good idea, changed.